website/docs/02 - Connection.md

```diff
@@ -89,59 +92,6 @@ const makeConnection = async (): Connection => {
 export default makeConnection;
 ```
 
-## Deploying MVOM database server features
```

Comment:
While there is no longer a `deployFeatures` method on the `Connection` class, there is still a `deploy` method on the class and deployment is still something that the consumer of the `Connection` class must manually do.  If the DB server deployment isn't valid, the `open` method will throw and a connection won't be opened.

Basically most of this section is still pertinent and just needs some tweaks.